addon/styles/_frost-text.scss

```diff
@@ -15,7 +15,6 @@ $padding: 0 19px 0 6px;
 .frost-text {
   display: flex;
   align-items: center;
-  width: 175px;
```

Comment:
Offline conversation:

@sglanzer To clarify my comment, if you remove the explicit `width`, the field will still ​_have_​ a width, it just won’t be explicit, so it’ll either go to browser default or it’ll go to 100% width.  Basically, I didn’t see any harm in making it our default

@job13er: ah, I see, so you're saying, chances are you'll need to override it regardless, so why not make it consistent if you don't